HW-Week2_Day03_Inheritance_and_ClassTypes.iml

```diff
@@ -1,12 +1,79 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" inherit-compiler-output="true">
-    <exclude-output />
-    <content url="file://$MODULE_DIR$">
-      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-    </content>
-    <orderEntry type="inheritedJdk" />
-    <orderEntry type="sourceFolder" forTests="false" />
-    <orderEntry type="library" name="KotlinJavaRuntime" level="project" />
-  </component>
-</module>
\ No newline at end of file
+//عملنا كلاس وخليناهopen نورث منو ونعمل  2-properties
+  open class Shape(var shapeName:String,var shapeColor:String){
+// عملنا funوسميناهاinfo وترجع two partmaterهيا shaper name,shape color
+       fun info(): String {
```

Comment:
why did you use open? what's the importance of it ? why the other once aren't all open explain your code

Reply:
to make inheritance from  shape